src/cms/widgets/LocalizedString/control.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from 'react'
+import * as R from 'ramda'
+
+import { getTranslationDefault, upsertTranslation, Localized } from '../i18n'
+import { Locale } from '../i18n-locales'
+
+interface WidgetProps<T> {
+  value: T
+  onChange: (value: T) => void
+  classNameWrapper?: string
+}
+
+interface LocalizedStringWidgetProps
+  extends WidgetProps<readonly Localized<string>[]> {}
+
+interface InputFieldProps extends LocalizedStringWidgetProps {
+  locale: Locale
+}
+
+const InputField: React.FunctionComponent<InputFieldProps> = ({
```

Comment:
Reorder this part to the bottom of the file so one starts with the important function `createLocalized...`

Reply:
Done.